webstr/WebSTR.py

```diff
@@ -0,0 +1,3 @@
+    parser.add_argument("--port", help="Port to run app", type=int, default=int(os.environ.get("FLASK_PORT", "5000")))
+    parser.add_argument("--port", help="Port to run app", type=int, default=int(os.environ.get("FLASK_PORT", "5000")))
+    parser.add_argument("--port", help="Port to run app", type=int, default=int(os.environ.get("FLASK_PORT", "5000")))
```

Comment:
Is this just duplicated code? I am also thinking if you could maybe rename this folder from webstr to something else just to not confuse things?

Reply:
maybe it can be called something like "container-deployment" and all other related Docker files can be in there? or docker-launcher